meinberlin/apps/projects/views.py

```diff
@@ -360,6 +360,7 @@ def project(self):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context["events"] = self.project.offlineevent_set.all().order_by("date")
+        return models.ProjectInsight.update_context(self.project, context)
```

Comment:
you mean `self.model here`?
when using a `classmethod` in the models.py, this will become `self.model.update_context(context)`